pkg/meta/version.go

```diff
@@ -2,7 +2,7 @@ package meta
 
 const (
 	// CLIAPIVersion used to communicate with user e.g. longhorn-manager
-	CLIAPIVersion    = 4
+	CLIAPIVersion    = 5
```

Comment:
Could you please elaborate more on when to bump this version?

Reply:
if the engine binary command definition used by longhorn-manager changes, we need to bump up CLIAPIVersion at least. If we don't allow the longhorn manager to use the old binary to talk with the old engines, we need to bump up CLIMinAPIVersion as well. Here, `don't allow` means there is no compatible code in the longhorn manager part: e,g, https://github.com/shuo-wu/longhorn-manager/blob/4368a4ac6521b267f756276a41e23e75b47f24a6/engineapi/backups.go#L260-L264

Ideally, CLIMinAPIVersion should be increased as long as the new longhorn manager cannot use the old binary without the compatible code. But here I don't want to block the live upgrade. The above ugly compatible code is introduced to avoid CLIMinAPIVersion change.

Reply:
Thanks for the info.